.circleci/unittest/linux/scripts/run-clang-format.py

```diff
@@ -122,13 +113,12 @@ def run_clang_format_diff_wrapper(args, file):
     except DiffError:
         raise
     except Exception as e:
-        raise UnexpectedError('{}: {}: {}'.format(file, e.__class__.__name__,
-                                                  e), e)
+        raise UnexpectedError("{}: {}: {}".format(file, e.__class__.__name__, e), e)
```

Comment:
f-string?

Reply:
If we want this everywhere, there is [`flynt`](https://pypi.org/project/flynt/) to do this conversion automatically. We could also add it to our auto formatters to enforce it.

Reply:
Got it, thanks!